waiter/src/waiter/statsd.clj

```diff
@@ -234,10 +233,9 @@
       "Publishes a Statsd timer metric for each of the 50th, 75th, 95th,
       and 100th percentiles of the values that have been locally aggregated"
       [metric-group metric values]
-      (dorun
-        (map (fn [[k v]]
-               (gauge #(metric-path metric-group (str metric "_p" k)) v))
-             (percentiles values 50 75 95 99 100))))
+      (run! (fn [[k v]]
```

Comment:
I think it's more clear to use `doseq` rather than `run!` + an anonymous `fn`, but I guess kibit's heuristics are just looking for `(dorun (map ....))`.

Reply:
There is a performance implication here as `(do (map ...))` will return a sequence with the return values of invoking the nested function. `run!` just returns `nil`.

Reply:
Right. This is a good change—I just think `doseq` would be even better. But I know these are automatic from kibit. I'm not asking you to change it.